src/reductiveContext.rei

```diff
@@ -0,0 +1,19 @@
+module type Config = {
+  type state;
+  type action;
+  let store: Reductive.Store.t(action, state);
+};
+module Make:
+  (Config: Config) =>
+   {
+    module Provider: {
+      [@bs.obj]
+      external makeProps:
+        (~children: 'children, ~key: string=?, unit) =>
+        {. "children": 'children} =
+        "";
+      let make: {. "children": React.element} => React.element;
+    };
+    let useSelector: (Config.state => 'a) => 'a;
```

Comment:
Where possible it'd be great to use more specific type variable names since they show up in editor tooling. Maybe something like `'selectedState` would be appropriate here?